agent/util-scripts/ara.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
There is no need for two playbooks, I modified ara.yml to run the tasks to generate results for localhost as well.